pkg/render/logstorage.go

```diff
@@ -674,17 +708,17 @@ func (es elasticsearchComponent) eckOperatorStatefulSet() *appsv1.StatefulSet {
 							},
 							Requests: corev1.ResourceList{
 								"cpu":    resource.MustParse("100m"),
-								"memory": resource.MustParse("20Mi"),
+								"memory": resource.MustParse("50Mi"),
 							},
 						},
 						Ports: []corev1.ContainerPort{{
-							ContainerPort: 9876,
+							ContainerPort: 9443,
```

Comment:
Just wanted to double check this change is correct

Reply:
ECK seems to have changed since 0.9.0 in commit 54c3f4a35dab7853fda4fd952bd812b71bd1eea3 . If we want things to be the same we can revert this line and also change it in the webhook service that is defined in this file. I am fine either way.